qa/suites/fs/upgrade/nofs/tasks/1-upgrade.yaml

```diff
@@ -23,7 +23,7 @@ tasks:
     - ceph fs dump
 - install.upgrade:
     mon.a:
-      branch: reef
+      branch: main
```

Comment:
```suggestion
```

Suggest removing this. We want to upgrade to whatever sha1 is specified for the test, not `main`.